client/src/context/useAuthContext.tsx

```diff
@@ -50,7 +50,7 @@ export const AuthProvider: FunctionComponent = ({ children }): JSX.Element => {
         } else {
           // don't need to provide error feedback as this just means user doesn't have saved cookies or the cookies have not been authenticated on the backend
           setLoggedInUser(null);
-          history.push('/login');
+          history.push('/land');
```

Comment:
```suggestion
          history.push('/landing-page');
```
Is this a typo or do we have another page for the '/land' route? 😄

Reply:
I was testing something😬